common/src/main/java/com/microsoft/identity/common/internal/net/HttpRequest.java

```diff
@@ -24,72 +24,34 @@
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import androidx.arch.core.util.Function;
+
+import com.microsoft.identity.common.java.net.HttpConstants;
```

Comment:
>com.microsoft.identity.common.java.net.HttpConstants [](http://example.com/codeflow?start=7&length=52)

this is interesting.. Why not just use  com.microsoft.identity.common.HttpConstants.. 
Are we also following the folder structure as per this pacakge name.

Reply:
That one will eventually be retired (once we're ready for a breaking change).

Basically, I'll use the code in common.java for anything that doesn't touch the interface.